hmda/src/main/scala/hmda/validation/dsl/PredicateCommon.scala

```diff
@@ -64,6 +66,7 @@ object PredicateCommon {
 
   def empty[A]: Predicate[A] = (_: A) match {
     case s: String => s.isEmpty
+    case s: EthnicityEnum => s.code == 0
```

Comment:
Our code uses custom comparators for validation, so I added a comparator here to reflect when ethnicity is blank.